xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -895,13 +895,15 @@ core.menu.create.comment=Comment to Page
 core.menu.create.attachment=Attachment to Page
 core.menu.copy=Copy
 core.menu.edit=Edit
+core.menu.edit.page=Edit page
```

Comment:
I meant reusing the "core.menu.edit" key instead of adding a new one. 

In the end, we used: 
core.menu.edit=Edit
core.menu.create=Create
core.menu.actions=Actions
core.menu.moreactions=More actions

as tooltips for the content menu.